tools/docker/static-analysis/suppressions.txt

```diff
@@ -3,3 +3,6 @@ uninitMemberVar
 
 // Suppress performance: Variable 'X' is assigned in constructor body. Consider performing initialization in initialization list.
 useInitializationList
+
+// Suppress style: Consider using std::find_if algorithm instead of a raw loop.
```

Comment:
I agree with you, but think it should be raised in the democracy channel

Reply:
Actually it is written on our coding convention to prefer ranged-loop. So this is not required.